app/scripts/components/pill/pill.tsx

```diff
@@ -4,20 +4,29 @@ import {useState} from 'react';
 import {CheckedIcon} from '../checkedIcon/checkedIcon';
 import {CloseIcon} from '../closeIcon/closeIcon';
 import {DEFAULT_PROJECT_PAGE_NAME} from '../../constants/constants';
+import {useDispatch, useSelector} from 'react-redux';
+import {clearSelectedPageId, setSelectedPageId} from '../../redux/reducers/selectedPage/selectedPage.actionCreator';
 
-export const Pill = ({pageName = DEFAULT_PROJECT_PAGE_NAME}) => {
-  const [isActive, setIsActive] = useState(false);
+export const Pill = ({pageName = DEFAULT_PROJECT_PAGE_NAME, pageId}: { pageName: string; pageId: number }) => {
   const [isRemove, setIsRemove] = useState(false);
-
+  // @ts-ignore
+  const {selectedPageId} = useSelector(state => state);
```

Comment:
Maybe it's better to use: `useSelector(state => state.selectedPageId, shallowEqual)`